Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
+                        let anotherBaseObject = ConcreteBaseObject()
 
-                    baseObject.listenTo(contextObject, eventName: eventName, callback: callback)
-                    contextObject.trigger(eventName)
+                        baseObject.listenTo(anotherBaseObject, eventName: eventName, callback: callback)
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == true
+                        expect(callbackWasCalled) == true
+                    }
                 }
-            }
 
-            describe("listenToOnce") {
-                it("Should fire callback just one time for an event on a given context object") {
-                    let contextObject = BaseObject()
+                describe("listenToOnce") {
+                    it("executes callback function just one time for an event on a given context object") {
+                        let anotherBaseObject = ConcreteBaseObject()
 
-                    baseObject.listenToOnce(contextObject, eventName: eventName, callback: callback)
-                    contextObject.trigger(eventName)
+                        baseObject.listenToOnce(anotherBaseObject, eventName: eventName, callback: callback)
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == true
+                        expect(callbackWasCalled) == true
 
-                    callbackWasCalled = false
-                    contextObject.trigger(eventName)
+                        callbackWasCalled = false
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("off") {
-                it("Callback should not be called if removed") {
-                    let listenId = baseObject.on(eventName, callback: callback)
-                    baseObject.off(listenId)
-                    baseObject.trigger(eventName)
+                describe("off") {
+                    it("doesn't executes callback function if it is removed") {
+                        let listenId = baseObject.on(eventName, callback: callback)
+                        baseObject.off(listenId)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
-                }
-                it("Callback should not be called if removed, but the others should") {
-                    var anotherCallbackWasCalled = false
-                    let anotherCallback: EventCallback = { _ in
-                        anotherCallbackWasCalled = true
+                        expect(callbackWasCalled) == false
                     }
+                    it("doesn't execute callback if it is removed, but the others are called") {
```

Comment:
If you agree with my suggestions above, this it will change to: `ìt("doesn't execute callback").
I think that is better to create another it to hold the logic of call others callback, what do you think?